Client/WebFilters/ShieldStats/Adblock/AdBlockStats.swift

```diff
@@ -37,6 +37,40 @@ public class AdBlockStats: LocalAdblockResourceProtocol {
     parseBundledGeneralBlocklist()
     loadDownloadedDatFiles()
   }
+  
+  /// Checks the general and regional engines to see if the request should be blocked.
+  ///
+  /// - Note: This method is should not be synced on `AdBlockStatus.adblockSerialQueue` and the result is synced on the main thread.
+  func shouldBlock(requestURL: URL, sourceURL: URL, resourceType: AdblockEngine.ResourceType, callback: @escaping (Bool) -> Void) {
+    Self.adblockSerialQueue.async { [weak self] in
+      let shouldBlock = self?.shouldBlock(requestURL: requestURL, sourceURL: sourceURL, resourceType: resourceType)
```

Comment:
`let shouldBlock = self?.shouldBlock(....) == true`
This way you don't have to unwrap `?? false`

Same for all the calls to `regionalAdblockEngine?.shouldBlock`